tests/demo.test.js

```diff
@@ -1,15 +1,83 @@
-test('should login', async () => {
-  const page = (await browser.pages())[0];
-  await page.goto('https://the-internet.herokuapp.com/login');
+describe('Test Login Page', () => {
+  describe('Login page is constructed correctly', () => {
+    let page;
 
-  const usernameElement = await page.$('#username');
-  const passwordElement = await page.$('#password');
-  const submitElement = await page.$('button[type="submit"]');
+    beforeAll( async() => {
+      page = (await browser.pages())[0];
+      await page.goto('https://the-internet.herokuapp.com/login');
+    });
 
-  await usernameElement.type('tomsmith');
-  await passwordElement.type('SuperSecretPassword!');
-  await submitElement.click();
+    test('Page is availble', () => {
+      expect(page).toBeDefined();
```

Comment:
In case `page` would not be defined this would already fail in line [7](https://github.com/christian-bromann/testrunner-demo/pull/2/files#diff-1604b05c50f24d02f17f9444cc7072efR7)